### DISCUSSION TITLE
如何更新农民有教程吗？[Bug]

### DISCUSSION THREAD

**Discussion by haun2022** [Chia-Network/chia-blockchain#11451] [General]
### What happened?

Significant harvester protocol update: You will need to update your farmer and all your harvesters as this is a breaking change in the harvester protocol. The new protocol solves many scaling issues. In particular, the protocol supports sending delta changes to the farmer - so for example, adding plots to a farm results in only the new plots being reported. We recommend you update your farmer first

### Version

1.3.4

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by loppefaaret** [comment]
Download the latest release from chia.net/download/
Close down the Chia client
Run the latest release installer
